<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Con_Multi</class>
 <widget class="QMainWindow" name="Con_Multi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>823</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuraci√≥n Multiclasificador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="Informacion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>801</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Reglas">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Rules</string>
    </property>
    <widget class="QComboBox" name="Regla">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>EQUAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DIFFERENT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GREATER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SMALLER</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Etiq_Ref">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>60</width>
       <height>32</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Peso">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>62</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="Anadir_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Aceptar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Clasificador</string>
    </property>
    <widget class="QLineEdit" name="Direccion_Carga">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>291</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Classifier</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>23</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Anadir">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Tipo</string>
    </property>
    <widget class="QRadioButton" name="Cascada">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cascade</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Votacion">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Voting</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
